src/uct/ib/base/ib_device.c

```diff
@@ -416,16 +416,24 @@ ucs_status_t uct_ib_device_query(uct_ib_device_t *dev,
     return UCS_OK;
 }
 
+struct uct_ib_sigaction {
```

Comment:
1. document this struct - does it correspond to kernel ABI?
2. indent on column